matter/si91x/siwx917/BRD4338A/support/inc/sl_si91x_m4_ps.h

```diff
@@ -0,0 +1,68 @@
+/***************************************************************************/ /**
```

Comment:
Can you fix the comment header. Remove the / / at the end of this line and styler should place everything back. Same for the .c file.

Reply:
Done